Extension/src/LanguageServer/lmTool.ts

```diff
@@ -235,7 +235,10 @@ export class CppConfigurationLanguageModelTool implements vscode.LanguageModelTo
                 contextString += `The project targets the ${chatContext.targetArchitecture} architecture. `;
                 telemetryProperties["targetArchitecture"] = chatContext.targetArchitecture;
             }
-
+            if (chatContext.usedTestFrameworks?.length > 0) {
+                contextString += `The active document contains the content of the file '${currentDoc.fileName}' which is located in a directory structure where other files are including headers for the following C++ test frameworks: ${chatContext.usedTestFrameworks.join(', ')} . `;

```

Comment:
Can this string be simplified to something like "The project uses these test frameworks: "?

Reply:
@sean-mcmanus let's discuss offline about this.